src/routes/route-factory.js

```diff
@@ -27,8 +28,20 @@ module.exports = function createRoute(repository) {
       .catch(next);
   }
 
+  function parseQueryParams(request, response, next) {

```

Comment:
This function is a middleware you can add it in the `src/middlewares` folder , please add unit tests also